src/component/actions/SetContactAttrib/SetContactAttribForm.test.tsx

```diff
@@ -0,0 +1,162 @@
+import { object } from 'prop-types';
+import * as React from 'react';
+import { SetContactField, SetContactProperty } from '../../../flowTypes';
+import { createStore } from '../../../store';
+import { createSetup } from '../../../testUtils';
+import ConnectedAttribElement from '../../form/AttribElement';
+import ConnectedTextInputElement from '../../form/TextInputElement';
+import { setContactField, setContactProperty } from './__test__';
+import {
+    fieldToSearchResult,
+    newFieldAction,
+    newPropertyAction,
+    propertyToSearchResult
+} from './helpers';
+import SetContactAttribForm, {
+    ATTRIB_HELP_TEXT,
+    SetContactAttribFormProps,
+    TEXT_INPUT_HELP_TEXT
+} from './SetContactAttribForm';
+
+const config = require('../../../../assets/config');
+
+const store = createStore();
+
+const context = {
+    endpoints: config.endpoints,
+    store
+};
+
+const childContextTypes = {
+    store: object
+};
+
+const baseProps: SetContactAttribFormProps = {
+    action: setContactProperty,
+    onBindWidget: jest.fn(),
+    updateAction: jest.fn()
+};
+
+const setup = createSetup<SetContactAttribFormProps>(
+    SetContactAttribForm,
+    baseProps,
+    context,
+    childContextTypes
```

Comment:
enzyme necessity, have to add a store to context and include childContextTypes when mounting a component that renders connected components